lib/exercises.rb

```diff
@@ -1,20 +1,67 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n log n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
-  raise NotImplementedError, "Method hasn't been implemented yet!"
-end
+# create an empty hash to store your word
+  hash = {}
+
+  #loop through strings
+  strings.each do |word|
+    key = word.chars.sort.join
+    #check if key is in the hash, push the word into the hash
+    #else, set the key as the word
+    if hash.key?(key)
```

Comment:
Just a bit more compact.
```suggestion
    if hash[key]
```